callisto/delivery/views.py

```diff
@@ -111,6 +111,7 @@ def submit_to_school(request, report_id, form_template_name="submit_to_school.ht
             except Exception:
                 # report was sent even if confirmation email fails, so don't show an error if so
                 logger.exception("couldn't send confirmation to user on submission")
+                context.update({'form': form, 'email_confirmation_error': True})
```

Comment:
I'd rather not add extra context to the response solely for tests. in the test, can we confirm that two things have happened: 1) the report was sent to school (best way to do this is probably with another mock) and 2) the confirmation template is the template that's returned?